day03/ex01/ex01.md

```diff
@@ -2,20 +2,19 @@
 
 |                         |                    |
 | -----------------------:| ------------------ |
-|   Turn-in directory :   |  ex01              |
-|   Files to turn in :    |  ImageProcessor.py |
-|   Forbidden functions : |  None              |
-|   Helpful libraries :   |  Matplotlib        |
+|   Turn-in directory:    |  ex01              |
+|   Files to turn in:     |  ImageProcessor.py |
+|   Forbidden functions:  |  None              |
+|   Helpful libraries:    |  Matplotlib        |
 
 Build a tool that will be helpful to load and display images in the upcoming exercises.
 
 Write a class named `ImageProcessor` that implements the following methods:
-* `load(path)` : opens the .png file specified by the `path` argument and returns an array with the RGB values of the image pixels.  
+* `load(path)` : opens the .png file specified by the `path` argument and returns an array with the RGB values of the image pixels
 It must display a message specifying the dimensions of the image (e.g. 340 x 500).
-
 * `display(array)` : takes a NumPy array as an argument and displays the corresponding RGB image.
```

Comment:
there is a dot here at the end and not in the previous element (not really important)

Reply:
I removed it on purpose to keep full stops only at last elements in list. But I can add them again in the next PR. We can align about some formatting guidelines for all bootcamps and maybe write some formatting guidelines in the wiki. What do you think?

Reply:
it is ok for now. i think we can think later to have the same ponctuation